app/js/app.js

```diff
@@ -211,6 +211,16 @@ ddsApp.run(function($rootScope, $state, $stateParams, $window, $modalStack, $anc
         }
     });
 
+    $rootScope.$on('$stateChangeSuccess', function () {
+        // Default - set page focus to h1
+        $timeout(function() {
```

Comment:
This is to add it to the next angular digest cycle, waiting for the page to load before executing the call.